drivers/mcp2515/candev_mcp2515.c

```diff
@@ -156,11 +156,6 @@ static int _send(candev_t *candev, const struct can_frame *frame)
     int ret = 0;
     enum mcp2515_mode mode;
 
-    if (frame->can_id > 0x1FFFFFFF) {
```

Comment:
Got any source backing up that this is valid and/or marks an extended ID? Because IIRC extended ID's were already supported.

(Ps. typo in the commit message, but I can't comment on that :) )

Reply:
Yes they are supported, which why I had issues with this check, you can find the definitions at: 

https://github.com/RIOT-OS/RIOT/blob/5643b528790cd74753fad2dd8db2232719cae820/sys/include/can/can.h#L51-L59

And code making reference to this in the driver code itself:

https://github.com/RIOT-OS/RIOT/blob/5643b528790cd74753fad2dd8db2232719cae820/drivers/mcp2515/candev_mcp2515.c#L460-L465

https://github.com/RIOT-OS/RIOT/blob/5643b528790cd74753fad2dd8db2232719cae820/drivers/mcp2515/mcp2515.c#L169-L178